src/components/SideBar.jsx

```diff
@@ -0,0 +1,97 @@
+// import React, { useState } from 'react';
+import {
+  FaTh,
+  FaUserAlt,
+  FaSignInAlt,
+} from 'react-icons/fa';
+import { NavLink } from 'react-router-dom';
+import Styles from './sidebar.module.css';
+
+const Sidebar = () => {
+  const menuItem = [
+    {
+      path: '/',
+      name: 'Home',
+      icon: <FaTh />,
+      end: true,
+
+    },
```

Comment:
I like the strategy of putting all the links in the object so that we can loop through it later. Good job!